staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -91,7 +91,7 @@ func TestApplyStripsFields(t *testing.T) {
 			"uid": "b",
 			"resourceVersion": "b"
 		}
-	}`), false)
+	}`), "apply", false)
```

Comment:
"fieldmanager_test"?

(people look at these as examples so try to give them a good lesson)